src/handlers/postToPRHandler.js

```diff
@@ -0,0 +1,23 @@
+const { Github } = require('../connectors');
+const config = require('../config');
+
+const github = Github();
+const enterpriseHost = config.get('github:domain');
+const ghUrl = enterpriseHost || 'https://www.github.com';
+
+module.exports = async function postReleaseVersionToPR({ owner, repo, number, head }) {
+  const body = `:rocket:   Posted by [**Captain's Log**](https://target.github.io/captains-log/#/):
```

Comment:
Would it make sense to pull this into a separate file to make it easy to edit/update as needed (maybe as a function so you can pass it the context you might want to include in the body)?

Reply:
Definitely, or I could do something with another config value like we use for release messaging. Could use a tagged template literal to interpolated the release link and version into that message as well. Good idea!

Reply:
Added: https://github.com/target/captains-log/pull/96/commits/a9dd19f48a8a9084740a4650dc002573da1c88cb